docs/UserGuide.md

```diff
@@ -219,24 +219,24 @@ Format: `list`
 * Displays student's email that is a clickable link to open the user's mailbox with a new email to the student's email address.
 
 
-### 7.4 Editing a student : `edit`
+### 7.4 Editing a person : `edit`
```

Comment:
i think it would be good to change it to student

Reply:
Yes, it seems that i did the merge conflict incorrectly. Please give me a moment